guides/common/modules/proc_synchronizing-a-content-view-to-a-smartproxy.adoc

```diff
@@ -0,0 +1,26 @@
+[id="Synchronizing_a_Content_View_to_a_{smart-proxy-context}{context}"]
+= Synchronizing a content view to a {SmartProxy}
+
+You can use the Hammer CLI to synchronize a content view to a {SmartProxy}.
+You need to specify the content view or content view ID along with the lifecycle environment or lifecycle environment ID.
+You will need to include the organization or organization ID as seen in the following examples:
+[options="nowrap", subs="+quotes,attributes"]
+----
+# hammer capsule content synchronize --id 1 \
```

Comment:
Please either drop duplicate commands or split them into individual code blocks so that there's only one command per code block.